app/Services/Manager.php

```diff
@@ -254,6 +254,32 @@ public function appendReportback($data, $parameters)
     }
 
     /**
+     * Get the top three reportbacks from a given leaderboard.
+     *
+     * @param array $leaderboard
+     * @return array $topThree
+     */
+    public function getTopThreeReportbacks($leaderboard)
+    {
+        $topThreeUsers = array_slice($leaderboard, 0, 3);
+        $places = ['1st', '2nd', '3rd'];
+
+        foreach ($topThreeUsers as $key => $user) {
+            $latestReportback = array_pop($user->reportback->reportback_items->data);
+
+            $topThree[] = [
+                'place' => $places[$key],
+                'first_name' => $user->first_name,
+                'quantity' => $user->reportback->quantity,
+                'image_url' => $latestReportback->media->uri,
+                'caption' => $latestReportback->caption,
+            ];
+        }
+
+        return $topThree;
```

Comment:
@weerd yea no errors. It's a php 5 thing. You can use square bracket syntax to create, modify arrays. 

> If $arr doesn't exist yet, it will be created, so this is also an alternative way to create an array.